dashboard/app/controllers/api/v1/pd/workshop_organizer_survey_report_controller.rb

```diff
@@ -0,0 +1,30 @@
+require 'controllers/api/v1/pd/workshop_score_summarizer'
+
+class Api::V1::Pd::WorkshopOrganizerSurveyReportController < Api::V1::Pd::ReportControllerBase
+  include WorkshopScoreSummarizer
+  load_resource :workshop, class: 'Pd::Workshop'
```

Comment:
There's no resource to load/auth here - I'm going to just bail if not a workshop_organizer